package.json

```diff
@@ -34,10 +34,10 @@
     "typescript": "^3.0.1"
   },
   "dependencies": {
-    "codechain-keystore": "^0.6.3",
-    "codechain-primitives": "^1.0.4",
     "commander": "^2.17.1",
     "enquirer": "^2.3.1",
+    "foundry-keystore-test": "^0.1.0",
```

Comment:
Why the library's name is foundry-keystore-test? The library is for temporary use?

Reply:
It was a mistake. I fixed it. Thank you : )